website/docs/providers/index.html.markdown

```diff
@@ -138,6 +138,8 @@ down to see all providers.
     </tr>
     <tr>
     <td><a href="/docs/providers/vsphere/index.html">VMware vSphere</a></td>
+    <td><a></a></td>
+    <td><a></a></td>
```

Comment:
Now you've drawn my attention to this bit I see that there's a missing `</tr>` here too. Technically that's not needed since the HTML parsing rules call for it to be generated implicitly here, but I believe it's required by the HTML conformance rules.

Reply:
yeah, i was having some weird issue with it before now after adding the blank spots was able to add the `</tr>` in the commit below.